ui/app/pages/error/error.component.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+class ErrorPage extends Component {
```

Comment:
I've been hesitant to default to using `PureComponent`, as it can end up doing more harm than good in many cases 🤔.

It's probably appropriate in this case though. I'll update this, and the other case.